mappings/net/minecraft/unmapped/C_cjyzmwwv.mapping

```diff
@@ -0,0 +1,17 @@
+CLASS net/minecraft/unmapped/C_cjyzmwwv
```

Comment:
#could you please name this class? I would go with matching the mojang name, `BanNoticeScreen`

Reply:
sure, if you don't mind I would prefer moving it to net.minecraft.client.multiplayer.report.BanNoticeScreen instead of net.minecraft.client.gui.screen.BanNoticeScreen though since it doesn't extend net.minecraft.client.gui.screen.Screen (unlike an actual screen)

Reply:
oh, strange!
I didn't actually look at the code, so I didn't realise it wasn't a real screen
in that case, there's probably a better name out there

Reply:
it's still used to *create* one though which makes it server the purpose of a screen.. I'm not sure either.